### ISSUE TITLE
In xCAT documentation, lots of WARNING for duplicate label

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#1098]
Seeing lots of WARNING messages like the following: 

```
../manage_clusters/common/deployment/deploy_os.rst:4: WARNING: duplicate label deploy_os, other instance in ../../docs/source/guides/admin-guides/manage_clusters/common/deployment/deploy_os.rst
../manage_clusters/common/deployment/deploy_os.rst:4: WARNING: duplicate label deploy_os, other instance in ../../docs/source/guides/admin-guides/manage_clusters/ppc64le/diskful/deploy_os.rst
```

This issue might provide some insight on how to fix it.  The include files or the label files should probably not be .rst extension.  https://github.com/sphinx-doc/sphinx/issues/1668

Or we should just get rid of the labels, it's not providing much value i don't think.

**Comment by daniceexi** [comment]
@whowutwut from the sphinx#1668 issue that looks like make include file to use non .rst as postfix could fix the duplicated label issue.
There are still a lot of warnings when building of xCAT doc, we can fix all of them together. 

Since this issue does not impact any real documentation pages, I moved this bug to 2.12.1


~~~~
/code/xcat-core.daniceexi/docs/source/advanced/docker/dockerized_xcat/run_xcat_in_docker_compose.rst:150: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/advanced/domain_name_resolution/domain_name_resolution.rst:198: WARNING: Literal block expected; none found.
/code/xcat-core.daniceexi/docs/source/advanced/index.rst:4: WARNING: toctree contains reference to nonexisting document u'advanced/webservices/index'
/code/xcat-core.daniceexi/docs/source/advanced/networks/infiniband/mlnxofed_ib_known_issue.rst:16: WARNING: Inline literal start-string without end-string.
/code/xcat-core.daniceexi/docs/source/advanced/restapi/restapi_resource/restapi_reference.rst:1536: WARNING: Literal block ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/acc_initrd_rootimg_gen.rst:61: WARNING: duplicate label pigz_example, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/diskless/customize_image/acc_initrd_rootimg_gen_ppc64le.rst
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/deploy_os.rst:4: WARNING: duplicate label deploy_os, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/diskless/deploy_os.rst
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst:94: WARNING: Title underline too short.

Support nodes in synclist file
~~~~~~~~~~~~~~~~~
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst:94: WARNING: Title underline too short.

Support nodes in synclist file
~~~~~~~~~~~~~~~~~
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/config_environment.rst:21: WARNING: duplicate label setup-dhcp, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/switch_discovery.rst
source/guides/admin-guides/manage_clusters/ppc64le/discovery/config_environment.rst:21: WARNING: duplicate label setup-dhcp, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/config_environment.rst
source/guides/admin-guides/manage_clusters/ppc64le/discovery/config_environment.rst:21: WARNING: duplicate label setup-dhcp, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/seq_discovery.rst
source/guides/admin-guides/manage_clusters/common/deployment/deploy_os.rst:4: WARNING: duplicate label deploy_os, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/deploy_os.rst
source/guides/admin-guides/manage_clusters/common/deployment/acc_initrd_rootimg_gen.rst:61: WARNING: duplicate label pigz_example, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/acc_initrd_rootimg_gen.rst
source/guides/admin-guides/manage_clusters/common/deployment/deploy_os.rst:4: WARNING: duplicate label deploy_os, other instance in /code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/diskful/deploy_os.rst
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/buildkit.1.rst:26: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/buildkit.1.rst:317: WARNING: Inline strong start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/buildkit.1.rst:317: WARNING: Inline strong start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/getadapter.1.rst:32: WARNING: undecodable source characters, replacing with "?": 'Traditionally, network interfaces in Linux are enumerated as eth[0123>>>\x85<<<], but these names do not necessarily correspond to actual labels on the chassis. \\ **getadapter**\\  help customer to get predictable network device name and some other network adapter information before provision or network configuration.'
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/getmacs.1.rst:44: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/lsve.1.rst:71: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/lsve.1.rst:73: ERROR: Unexpected indentation.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/lsve.1.rst:74: WARNING: Block quote ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/rmkitcomp.1.rst:32: WARNING: Inline strong start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/rmkitcomp.1.rst:32: WARNING: Inline strong start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/unregnotif.1.rst:52: WARNING: Inline strong start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/unregnotif.1.rst:52: WARNING: Inline strong start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man1/xdcp.1.rst:252: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:49: WARNING: Block quote ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:51: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:62: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:73: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:73: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:74: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:107: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:118: WARNING: Block quote ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/nics.5.rst:124: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/noderes.5.rst:58: WARNING: Block quote ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/prescripts.5.rst:50: ERROR: Unexpected indentation.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/prescripts.5.rst:50: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/prescripts.5.rst:51: WARNING: Block quote ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/prescripts.5.rst:57: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/prescripts.5.rst:59: ERROR: Unexpected indentation.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man5/vm.5.rst:139: WARNING: Definition list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man7/boottarget.7.rst:22: WARNING: Inline strong start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man7/group.7.rst:624: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man7/node.7.rst:624: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/tabprune.8.rst:22: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/winstall.8.rst:60: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/winstall.8.rst:66: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/winstall.8.rst:72: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:40: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:116: WARNING: Inline substitution_reference start-string without end-string.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:134: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:139: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:144: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:149: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:154: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/references/man8/xcatdebug.8.rst:159: WARNING: Enumerated list ends without a blank line; unexpected unindent.
/code/xcat-core.daniceexi/docs/source/advanced/chain/create_image_for_runimage.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/hamn/setup_ha_mgmt_node_with_drbd_pacemaker_corosync.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/hamn/setup_ha_mgmt_node_with_raid1_and_disks_move.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/hamn/setup_ha_mgmt_node_with_shared_data.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/hamn/setup_xcat_high_available_management_node_in_softlayer.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/hierarchy/databases/postgres_tips.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/hierarchy/define_and_install_compute_node.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/hierarchy/define_service_node.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/networks/getadapter.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/networks/vlan/index.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/developers/doc_guidelines.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/developers/license/xcat_corporate_contributor_license_agreement.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/developers/license/xcat_individual_contributor_license_agreement.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/acc_initrd_rootimg_gen.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/cfg_network.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/cfg_partition.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/cfg_second_adapter.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/create_img.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/deploy_os.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/driver_update_disk.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/enable_kdump.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/generate_img.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/install_new_kernel.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/raid_cfg.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/trim_diskless_rootimg.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/discover/manually_discovery.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/discover/seq_discovery.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/discover/switch_discovery.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/kvm/manage_vm.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/rbeacon.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/rcons.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/renergy.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/reventlog.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/rflash.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/rinv.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/rpower.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/rspconfig.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/management/rvitals.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/parallel_cmd.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/updatenode.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/config_environment.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/pbmc_discovery.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/pbmc_discovery_with_bmcdiscover.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/pbmc_discovery_with_lsslp.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/schedule_environment.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/standard_cn_definition.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/diskless/customize_image/cfg_partition.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/diskless/customize_image/raid_cfg.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/install-guides/common/uninstall_xcat.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/guides/install-guides/common_sections.rst:: WARNING: document isn't included in any toctree
/code/xcat-core.daniceexi/docs/source/advanced/domain_name_resolution/domain_name_resolution.rst:473: WARNING: undefined label: confignics_label (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/advanced/hierarchy/appendix/appendix_b_diagnostics.rst:10: WARNING: undefined label: setup_service_node_stateful_label (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/advanced/hierarchy/appendix/appendix_b_diagnostics.rst:10: WARNING: undefined label: setup_service_node_stateful_label (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/advanced/hierarchy/databases/mysql_remove.rst:15: WARNING: unknown document: /guides/admin-guides/large_clusters/databases/postgres_install
/code/xcat-core.daniceexi/docs/source/advanced/mixed_cluster/support_matrix.rst:49: WARNING: undefined label: building_a_stateless_image_of_a_different_architecture_or_os (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/advanced/networks/infiniband/switch_configuration.rst:77: WARNING: unknown document: ../../../guides/admin-guides/references/man/xdsh.1.html
/code/xcat-core.daniceexi/docs/source/advanced/sysclone/sysclone.rst:43: WARNING: undefined label: install_guides (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/developers/doc_guidelines.rst:106: WARNING: unknown document: /overview/support_list
/code/xcat-core.daniceexi/docs/source/developers/doc_guidelines.rst:112: WARNING: unknown document: ../overview/support_list
/code/xcat-core.daniceexi/docs/source/developers/guides/docs/doc_guidelines.rst:106: WARNING: unknown document: /overview/support_list
/code/xcat-core.daniceexi/docs/source/developers/guides/docs/doc_guidelines.rst:112: WARNING: unknown document: ../overview/support_list
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/basic_concepts/global_cfg/index.rst:12: WARNING: unknown document: /guides/admin-guides/references/man3/noderange
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/additionalpkg/ubuntu_os_other_pkg.rst:35: WARNING: undefined label: ubuntu-install-non-internet (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/deployment/install_new_kernel.rst:82: WARNING: unknown document: cfg_second_adapter.rst
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/common/updatenode.rst:97: WARNING: undefined label: using-the-mypostscript-template (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/switch_discovery.rst:22: WARNING: undefined label: predefined_switches_label (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/discovery/switch_discovery.rst:23: WARNING: undefined label: predefined_server_nodes_label (if the link has no caption the label must precede a section header)
source/guides/admin-guides/manage_clusters/common/deployment/install_new_kernel.rst:82: WARNING: unknown document: cfg_second_adapter.rst
source/guides/admin-guides/manage_clusters/common/updatenode.rst:97: WARNING: undefined label: using-the-mypostscript-template (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/guides/admin-guides/manage_clusters/ppc64le/virtual_machines/powerKVM.rst:9: WARNING: undefined label: diskfull_installation (if the link has no caption the label must precede a section header)
/code/xcat-core.daniceexi/docs/source/guides/install-guides/common/uninstall_xcat.rst:11: WARNING: unknown document: /guides/admin-guides/references/man/dumpxCATdb.1
/code/xcat-core.daniceexi/docs/source/guides/install-guides/common/uninstall_xcat.rst:228: WARNING: unknown document: /guides/admin-guides/large_clusters/databases/postgres_remove
/code/xcat-core.daniceexi/docs/source/guides/install-guides/common/uninstall_xcat.rst:229: WARNING: unknown document: /guides/admin-guides/large_clusters/databases/mysql_remove
/code/xcat-core.daniceexi/docs/source/guides/install-guides/common/uninstall_xcat.rst:238: WARNING: unknown document: /guides/admin-guides/references/man/restorexCATdb.1
WARNING: html_static_path entry u'/code/xcat-core.daniceexi/docs/source/_static' does not exist
~~~~

**Comment by gurevichmark** [comment]
I will try to get rid of as many errors and warnings as possible.

**Comment by gurevichmark** [comment]
Majority of problems was fixed by pull request #1201 
Remaining problems are 2 missing labels, 1 missing file and errors from man5 and man7 generated man pages.
Separate issues will be opened to fix them.

**Comment by whowutwut** [comment]
Thanks for the changes @gurevichmark  I think with the reduced warning messages it will be easier to spot the documentation errors
